packages/mobile/locales/es-419/fiatExchangeFlow.json

```diff
@@ -25,6 +25,7 @@
   "receiveOnAddress": "Dirección de cUSD/CELO",
   "receiveWithPonto": "GCash (con Ponto)",
   "receiveWithKotani": "M-PESA (con Kotani Pay)",
+  "receiveWithBidali": "Gift Cards and Mobile Top Up",
```

Comment:
Currently waiting for the actual translation. Will add it as soon as I receive it.